server/src/test/scala/services/export/StreamingFlightsExportSpec.scala

```diff
@@ -307,7 +307,7 @@ class StreamingFlightsExportSpec extends CrunchTestLike {
 
     val expected =
       s"""|$flightHeadings,$apiHeadings,$actualApiHeadings
-          |SA0324,SA0324,JHB,/,Expected,2017-01-01,20:00,20:00,,,,20:00,,100,Y,7,15,32,46,12,23,30,35,,,,,0.0,0.0,0.0,1.0,2.0,3.0,0.0,6.0,7.0,4.0,5.0,0.0
+          |SA0324,SA0324,JHB,/,Expected,2017-01-01,20:00,20:00,,,,20:00,28,28,,2,4,9,13,3,7,8,10,,,,,0.0,0.0,0.0,1.0,2.0,3.0,0.0,6.0,7.0,4.0,5.0,0.0
```

Comment:
Do we know that this changed test result is the desired behaviour? ie, is this because the flight in the test has no live arrival source so it's using the api data for pax nos?

Reply:
Yes  that is case here . `flightWithAllTypesOfAPISplitAndNoLiveNos` val has arrival with no actual pax number and result split for `ApiSplitsWithHistoricalEGateAndFTPercentages` split source